addons/sale_crm/crm_lead.py

```diff
@@ -11,6 +17,30 @@ def _get_sale_amount_total(self, cr, uid, ids, fields, args, context=None):
             res[key] = value
         return res
 
+    def action_view_quotation(self, cr, uid, ids, context=None):
+    # '''
+    # This function returns an action that display existing quotations of given opportunity ids. It can either be a in a list or in a form view, if there is only one quotation to show.
+    # '''
+        mod_obj = self.pool.get('ir.model.data')
+        act_obj = self.pool.get('ir.actions.act_window')
+
+        result = mod_obj.get_object_reference(cr, uid, 'sale', 'action_quotations')
+        id = result and result[1] or False
+        result = act_obj.read(cr, uid, [id], context=context)[0]
+        #compute the number of quotation to display
+        quote_ids = []
+        for op in self.browse(cr, uid, ids, context=context):
```

Comment:
Is'nt it easier to do quote_ids = self.pool['sale.order'].search(cr, uid, [('opportunity_id', 'in', ids)], context=context) ?